index.html

```diff
@@ -10,8 +10,190 @@
+            <li class="menu"><a href="#news" class="hover">News</a></li>
+          </ul>
+          <a class="button-nav">Europe Global Summit</a>
+        </div>
+      </header>
+      <div class="headings">
+        <p class="sub-heading">"Hello! Europe"</p>
+        <h1 class="main-text">Europe Global Summit 2023</h1>
+      </div>
+      <div class="border-text">
+        <p class="border-paragraph">
+          Europe Summit is the biggest gathering events in the Europe. It unites
+          all of the countries of Europe with providing speakers and
+          presentations from the rest of the World.
+        </p>
+      </div>
+      <div class="date">
+        <h2 class="date-heading">06.02.2023(MON) - 18(SAT)</h2>
+        <span class="date-span"
+          >@ Geneva, Switzerland, Geneva Center for the Arts</span
+        >
+      </div>
+    </section>
+    <section class="workshop" id="program">
+      <h4 class="h4-text">Main Program</h4>
+      <hr class="red-line" />
+      <div class="flex-program">
+        <div class="program">
+          <img
+            class="program-img"
+            src="images/lecture.png"
+            alt="lecture icon"
+          />
+          <h5 class="program-text">Lecture</h5>
+          <p class="program-paragraph">
+            Exploration of the effect of digital lecture supplementation.
+          </p>
+        </div>
+        <div class="program">
+          <img
+            class="program-img"
+            src="images/gallery.png"
+            alt="gallery-icon"
+          />
+          <h5 class="program-text">Exhibition</h5>
+          <p class="program-paragraph">
+            From real life into the world of art is a virtual exhibition.
+          </p>
+        </div>
+        <div class="program">
+          <img class="program-img" src="images/chat.png" alt="chat icon" />
+          <h5 class="program-text">Forum</h5>
+          <p class="program-paragraph">
+            The gaol of the forum is to show support of digital technologies.
+          </p>
+        </div>
+        <div class="program">
+          <img
+            class="program-img"
+            src="images/workshop.png"
+            alt="workshop icon"
+          />
+          <h5 class="program-text">Workshop</h5>
+          <p class="program-paragraph">
+            Set of tasks and goals to improve skills in a modern world.
+          </p>
+        </div>
+        <div class="program">
+          <img
+            class="program-img"
+            src="images/global-network.png"
+            alt="global-network icon"
+          />
+          <h5 class="program-text">Ignite</h5>
+          <p class="program-paragraph">
+            Opportunity to network with digital CEO from all over the world.
+          </p>
+        </div>
+      </div>
+      <button class="join" id="join">Join Europe Summit Now</button>
+      <span class="join-now">SEE THE WORLD PROGRAM</span>
+    </section>
```

Comment:
- [ ] You did a good job styling the Main Program section :+1:, but as a project requirement, you should have the exact same colors (#ec5242 instead of #ec5241), borders (each card should not have borders), and backgrounds from the original design.
- Hint: [Here](https://ingdavidtm.github.io/capstone-biotechnology/images/patternBackground.png) is the pattern for the background
- [ ] While you did a wonderful job creating all your articles :+1: , for semantics you should use an `article` tag for each element inside the "main program" section

**Your design**

![Screenshot from 2023-02-07 08-15-59](https://user-images.githubusercontent.com/105117832/217256160-25d26503-f281-42f5-ad9f-94b629b5e9d4.png)

**Template**

![Screenshot from 2023-02-07 08-16-43](https://user-images.githubusercontent.com/105117832/217256237-1d8c1b07-c289-4361-b29f-bb9204168c65.png)